pytorch_blade/benchmark/TorchBench/results_anaysis.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import pandas as pd
+
+COMPARE_FIELDS = [
+    "disc (latency)",
+    "blade (latency)",
+    "dynamo-blade (latency)",
+    "dynamo-disc (latency)",
+]
+RELATED_DIFF_PERCENT = 3
+GITHUB_ISSUE_TEMPLATE = """
+TorchBench CI has detected a performance signal.
+
+Affected Tests:
+{results}
+
+detail data can be seen in {oss_dir}
+created by TorchBench CI automatically
+"""
+
+
+def try_cast_to_float(to_cast: str):
+    try:
+        to_cast = float(to_cast)
+    except:
+        pass
```

Comment:
Nit: print the exception error message.

Reply:
some models cannot run currently, so it's rational that some results are like 'RuntimeError'， ’UnserializableException‘， we have handled it if status changed.

Reply:
OK.